packages/terra-search-field/lib/SearchField.scss

```diff
@@ -0,0 +1,46 @@
+@import '~terra-mixins';
+
+.terra-SearchField {
+  display: flex;
+  width: 100%;
+
+  .terra-SearchField-input {
+    @include terra-border-radius-start(0.286em);
+    @include terra-border-radius-end(0);
+
+    appearance: textfield;
+    border: 0.071em solid $terra-grey-15;
+    flex: 1 1 auto;
+    font-family: inherit;
+    font-size: 1em;
+    line-height: 1.143;
+    margin: 0;
+    min-width: 0;
+    padding: 0.286rem;
+    width: 100%;
+
+    &:focus {
+      border-color: $terra-blue-60;
+      outline: none;
+    }
+  }
+
+  .terra-SearchField-button {
```

Comment:
I think we need to have our own button class here because of how the search field and button need to kind of flow together (see screenshot).
<img width="588" alt="screen shot 2017-05-23 at 9 47 36 am" src="https://cloud.githubusercontent.com/assets/28159802/26360129/ea4a83f0-3f9c-11e7-86fe-4f3b2ce8cfc7.png">